lms/djangoapps/branding/__init__.py

```diff
@@ -17,12 +17,16 @@
 from openedx.core.djangoapps.content.course_overviews.models import CourseOverview
 
 
-def get_visible_courses():
+def get_visible_courses(org=None):
     """
     Return the set of CourseDescriptors that should be visible in this branded instance
     """
-    filtered_by_org = microsite.get_value('course_org_filter')
-    courses = CourseOverview.get_all_courses(org=filtered_by_org)
+    microsite_org = microsite.get_value('course_org_filter')
+
+    orgs = [org, microsite_org]
+    orgs = [org for org in orgs if org is not None]
+
+    courses = CourseOverview.get_all_courses(orgs=orgs)
```

Comment:
revising my earlier comment - it seems like the expected behavior is even simpler, since there are only two orgs to worry about - if they both have the same value, return results for that org - if not, return an empty result.  though in the test the behavior is something else - microsite always wins, even if the caller specified some other org to filter by.  doesn't that seem like it could lead to surprises?